site/templates/collection/carousel/item/buttons.jet

```diff
@@ -0,0 +1,26 @@
+{{import "./extras.jet"}}
+
+{{block carouselItemButtons()}}
+  {{slug := .Slug}}
+  {{isFilm := .ItemType == "film"}}
+  {{hasTrailer := (isset(.InnerItem["Trailers"]) && len(.InnerItem.Trailers) > 0) || (.InnerItem["PromoURL"] && len(.InnerItem.PromoURL) > 0)}}
+  {{letterboxdID = ""}}
```

Comment:
Interesting, this is using `=` not `:=` but I can't see where else the variable is declared. It looks like it's just assigning without declaring which I don't think is valid Go but maybe it's valid Jet?

Reply:
Might be valid in jet?  I reckon the `:` should be added in anyway.